src/org/traccar/protocol/TeltonikaProtocolDecoder.java

```diff
@@ -269,6 +363,8 @@ private void decodeLocation(Position position, ChannelBuffer buf, int codec) {
             }
         }
 
+        decodeNetwork(position);
```

Comment:
We can try decode Network only after decoding all IOs. Have to moved code to function, because method to long.